plugins/CoreHome/templates/_uiControl.twig

```diff
@@ -6,4 +6,8 @@
     data-params="{{ clientSideParameters|json_encode }}">
 {% render implView with implOverride %}
 </div>
-<script>$(document).ready(function () { require('{{ jsNamespace }}').{{ jsClass }}.initElements(); });</script>
\ No newline at end of file
+<script>
+    document.addEventListener('DOMContentLoaded', function () {
+        $(document).ready(function () { require('{{ jsNamespace }}').{{ jsClass }}.initElements(); });
+    }, false);
```

Comment:
Verified locally and via https://jsfiddle.net/9xnh71uy/ that we can have `$(document).ready` inside `DOMContentLoaded` event handler and it works as expected.